### PR TITLE
create card component and section best seller on main page

### PR DESCRIPTION
**Pull request by Nizar7zak**
relates #22

### CODE REVIEW

**Comment by Nizar7zak** [comment]
Best seller Section is responsive, check it please. because there is no data I add a fake data to cards to present it.

**Review by emanNahed — APPROVED** [review]

**Review by emanNahed — DISMISSED** [review]

**Review by muath-khatib — DISMISSED** [review]

**Review by aaamra — DISMISSED** [review]

File: client/src/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Fragment } from 'react';
+import BestSellerSection from './components/BestSellerSection';
+import Header from './components/Header';
+
+const Home = () => (
+  <Fragment>
+    <Header />
+    <BestSellerSection />
+  </Fragment>
```
Comment (Mu7ammadAbed) [inline client/src/Home.jsx:9]:
```suggestion
  <>
    <Header />
    <BestSellerSection />
  </>
```

File: client/src/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Fragment } from 'react';
```
Comment (Mu7ammadAbed) [inline client/src/Home.jsx:1]:
```suggestion
```

File: client/src/components/BestSellerSection.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <div className='best-seller-section'>
+        <h2 className='title-section'>Best Seller</h2>
+      <div className="seller">
+        <ProductCard />
+        <ProductCard />
+        <ProductCard />
+        <ProductCard />
+        <ProductCard />
+        <ProductCard />
```
Comment (Mu7ammadAbed) [inline client/src/components/BestSellerSection.jsx:15]: What is this ?

File: client/src/components/Card/ProductCard.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        style={{ width: '100%', borderRadius: 4, border: '1px solid #DDD', padding:1 }}
+        cover={
+          <img
+            style={{
+              maxWidth: '100%',
+              maxHeight: '100%',
+            }}
+            alt="example"
+            src="https://www.aroma-zen.com//cache/images/images/products/47/2787/2787_lait_corporel_regenerant_a_la_grenade_200_ml_1.jpg.thumb_1000x800.jpg"
+          />
```
Comment (Mu7ammadAbed) [inline client/src/components/Card/ProductCard.jsx:21]: You shouldn't pass it like this

**Comment by Mu7ammadAbed** [comment]
I think this PR is not ready yet

File: client/src/components/BestSellerSection.jsx (hunk 3 above)
Comment (Nizar7zak) [reply client/src/components/BestSellerSection.jsx:15]: this is a fake data i put it to check layout for entire section

**Comment by Nizar7zak** [comment]
> I think this PR is not ready yet

**Review by emanNahed — APPROVED** [review]

**Review by muath-khatib — APPROVED** [review]

**Review by reemsh2000 — APPROVED** [review]

**Review by aaamra — APPROVED** [review]

**Review by emanNahed — DISMISSED** [review]

**Review by emanNahed — APPROVED** [review]

**Review by reemsh2000 — APPROVED** [review]

File: client/src/Home.jsx (hunk 2 above)
Comment (Nizar7zak) [reply client/src/Home.jsx]: done!

File: client/src/components/Card/ProductCard.jsx (hunk 4 above)
Comment (Nizar7zak) [reply client/src/components/Card/ProductCard.jsx]: done!

**Review by muath-khatib — APPROVED** [review]

**Review by Mu7ammadAbed — APPROVED** [review]

**Pull request merged**